unit_tests/log_transfer/test_main_chkpt_info.cpp

```diff
@@ -0,0 +1,800 @@
+/*
+ * Copyright 2008 Search Solution Corporation
+ * Copyright 2016 CUBRID Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+#define CATCH_CONFIG_MAIN
+#include "catch2/catch.hpp"
+
+#include "client_credentials.hpp"
+#define private public
+#include "log_checkpoint_info.hpp"
+#undef private
+#include "log_impl.h"
+#include "log_lsa.hpp"
+#include "log_record.hpp"
+#include "mem_block.hpp"
+#include "thread_entry.hpp"
+#include "system_parameter.h"
+
+#include <algorithm>
+#include <cstdlib>
+#include <map>
+#include <random>
+#include <vector>
+
+using namespace cublog;
+
+class test_env_chkpt
+{
+  public:
+    test_env_chkpt ();
+    test_env_chkpt (int size_trans, int size_sysops);
+    ~test_env_chkpt ();
+
+    LOG_LSA generate_log_lsa();
+    LOG_TDES *generate_tdes (int index);
+    LOG_2PC_GTRINFO generate_2pc_gtrinfo();
+    LOG_2PC_COORDINATOR *generate_2pc_coordinator();
+    CLIENTIDS generate_client (int index);
+    LOG_INFO_CHKPT_TRANS generate_log_info_chkpt_trans();
+    LOG_INFO_CHKPT_SYSOP generate_log_info_chkpt_sysop();
+    std::vector<LOG_LSA> used_logs;
+    void generate_tran_table();
+
+    static constexpr int MAX_RAND = 32700;
+    static void require_equal (checkpoint_info before, checkpoint_info after);
+
+    checkpoint_info before;
+    checkpoint_info after;
```

Comment:
Make members private. Name them using the m_ suffix.
```suggestion
  private:
    checkpoint_info m_before;
    checkpoint_info m_after;
```